components/LeagueEntries/LeagueEntries.tsx

```diff
@@ -59,18 +59,27 @@ const LeagueEntries = ({
         >
           {userPickHistory?.map((logoURL, index) => (
             <div
-              key={logoURL}
+              key={`${logoURL ? logoURL : 'no-pick'}-${index + 1}`}
               className="flex flex-col h-[66px] opacity-80 items-center justify-center border border-border px-4 py-1 rounded-lg gap-1"
             >
               <span className="text-xs">WEEK {index + 1}</span>
-              <Image
-                className="league-entry-logo"
-                width={40}
-                height={40}
-                data-testid="league-history-logo"
-                src={logoURL}
-                alt="teamLogo"
-              />
+              {logoURL ? (
+                <Image
+                  className="league-entry-logo"
+                  width={40}
+                  height={40}
+                  data-testid="league-history-logo"
+                  src={logoURL}
+                  alt="teamLogo"
+                />
+              ) : (
+                <span
+                  className="text-xs h-10 w-full text-primary pt-2"
```

Comment:
I'm worried about the a11y of the color and the small text size, did you check this?

Reply:
@ryandotfurrer I did not check this.  hmmmmm @shashilo what do you think?

Reply:
Fine for now. A11Y audit can come after.